src/MongoDB.Driver.Core/Core/Operations/AggregateOperation.cs

```diff
@@ -468,6 +471,11 @@ public override AggregateResult Deserialize(BsonDeserializationContext context,
                             result.Results = arraySerializer.Deserialize(context);
                             break;
 
+                        case "postBatchResumeToken":
+                            var bsonSerializer = new BsonDocumentSerializer();
+                            result.PostBatchResumeToken = bsonSerializer.Deserialize(context);
```

Comment:
You can use the pre-recreated BsonDocumentSerializer:

    result.PostBatchResumeToken = BsonDocumentSerializer.Instance.Deserialize(context);

Reply:
Done